frontend/src/features/properties/components/GeocoderAutoComplete.scss

```diff
@@ -43,5 +43,6 @@
 }
 
 .GeocoderAutoComplete option:hover {
-  background-color: $accent-color;
+  background-color: $active-color;
+  color: #fff;
```

Comment:
Changes Geocoder dropdown highlight colour to blue so it matches other dropdowns styles